packages/paste-core/components/select/__tests__/option.test.tsx

```diff
@@ -74,4 +79,63 @@ describe('Form | Option', () => {
 
     expect(results).toHaveNoViolations();
   });
+
+  describe('HTML Attribute', () => {
+    it('should set an element data attribute for Option (default)', () => {
+      testRender(<MockOption suffix="default-data-attribute" />);
+      expect(screen.getByTestId('option-default-data-attribute').getAttribute('data-paste-element')).toEqual('OPTION');
+    });
+
+    it('should set an element data attribute for Option', () => {
+      testRender(<MockOption suffix="unique-data-attribute" element="UNIQUE_NAME" />);
+
+      expect(screen.getByTestId('option-unique-data-attribute').getAttribute('data-paste-element')).toEqual(
+        'UNIQUE_NAME'
+      );
+    });
+  });
+
+  describe('Customization', () => {
+    it('should add custom styles to Option', () => {
+      testRender(
+        <CustomizationProvider
+          // @ts-expect-error global test variable
+          theme={TestTheme}
+          elements={{
+            OPTION: {
+              color: 'colorTextLinkDarker',
+            },
+            OPTION_GROUP: {
+              color: 'colorText',
+            },
+          }}
+        >
+          <MockOption suffix="custom-styles" />
+        </CustomizationProvider>
+      );
+
+      expect(screen.getByTestId('option-custom-styles')).toHaveStyleRule('color', 'rgb(3,11,93)');
```

Comment:
It might make sense to test for both of the stylings included in the custom theme, to make sure that both OPTION and OPTION_GROUP are behaving as expected. Thoughts?

Reply:
Oh I think I see that this is being tested in the next spec, got it!